website/sidebars.js

```diff
@@ -11,51 +11,51 @@ module.exports = {
             'terminology',
         ],
         Tutorials: [
-          'tutorials/intro',
-          {
-            type: 'category',
-            label: 'Basic Tutorial',
-            items: [
-                {
-                    type: 'category',
-                    label: 'Your first Hydra app',
-                    items: [
-                        'tutorials/basic/your_first_app/simple_cli',
-                        'tutorials/basic/your_first_app/config_file',
-                        'tutorials/basic/your_first_app/using_config',
-                        'tutorials/basic/your_first_app/config_groups',
-                        'tutorials/basic/your_first_app/defaults',
-                        'tutorials/basic/your_first_app/composition',
-                    ]
-                },
-                {
-                    type: 'category',
-                    label: 'Running your Hydra app',
-                    items: [
-                        'tutorials/basic/running_your_app/multi-run',
-                        'tutorials/basic/running_your_app/working_directory',
-                        'tutorials/basic/running_your_app/logging',
-                        'tutorials/basic/running_your_app/debugging',
-                        'tutorials/basic/running_your_app/tab_completion',
-                    ]
-                },
+            'tutorials/intro',
```

Comment:
Don't reformat.
if you want to reformat it, create a different issue and also mention how to achieve the same formatting.